h/auth/__init__.py

```diff
@@ -41,6 +44,15 @@ def includeme(config):
     global DEFAULT_POLICY
     global WEBSOCKET_POLICY
 
+    config.register_service_factory('.services.auth_ticket_service_factory',
+                                    iface=pyramid_authsanity.interfaces.IAuthService)
```

Comment:
The interface can be a dotted string here so you can avoid importing `pyramid_authsanity` altogether:

```python
    config.register_service_factory('.services.auth_ticket_service_factory',
                                    iface='pyramid_authsanity.interfaces.IAuthService')
```